build.gradle

```diff
@@ -232,6 +242,7 @@ dependencies {
     implementation 'jakarta.annotation:jakarta.annotation-api:1.3.5'
     implementation "org.opensearch.plugin:transport-netty4-client:${opensearch_version}"
     implementation "org.opensearch.client:opensearch-rest-high-level-client:${opensearch_version}"
+    implementation "org.opensearch:common-utils:${common_utils_version}"
```

Comment:
This is scary. Common utils was defined to abstract out security implementation for other plugins.

I understand this is for testing only. 
Lets take it as a test dependency instead of regular dependency? 
Take a look at AD[1] for examples.

[1] https://github.com/opensearch-project/anomaly-detection/blob/9f6a5abb9652fb010decde8778c56c292eac90e2/build.gradle#L680

Reply:
done